crates/telio-wg/src/wg.rs

```diff
@@ -1819,4 +1816,38 @@ pub mod tests {
             ));
         }
     }
+
+    #[tokio::test(start_paused = true)]
+    async fn test_bytes_and_timestamps() {
+        let build = |rx_ts, tx_ts| BytesAndTimestamps {
+            rx_bytes: 5,
+            tx_bytes: 5,
+            rx_ts,
+            tx_ts,
+        };
+
+        let now = Instant::now();
+        let a_second_ago = now.checked_sub(Duration::from_secs(1)).unwrap();
+        let eleven_seconds_ago = now.checked_sub(Duration::from_secs(11)).unwrap();
+        let rtt = Duration::from_secs(0);
+
+        let missing_ts_1 = build(None, None);
```

Comment:
I think this might be simplified by using test parametrization from rstest crate, but this is just a suggestion so feel free to resolve this yourself if you think it's not worth the hassle to implement.